src/components/InputV2/PrivateKey.tsx

```diff
@@ -71,20 +72,24 @@ export const PrivateKeyInput = ({
  *
  * error: set by private key input validation
  * type: always "text" for private key input
+ * autoComplete: always "off" for private key
+ * placeholder: always "Enter your private key"
  *
- * We add "setIsValid"
+ * We add "setIsValid" and "importError"
  */
 
 export type PrivateKeyInputProps = {
   setIsValid?: (isValid: boolean) => void
   importError?: string
-} & Omit<BaseInputProps, 'error' | 'type' | 'placeholder'>
+} & Omit<BaseInputProps, 'error' | 'type' | 'autoComplete' | 'placeholder'>
 
-const { error, type, placeholder, ...addressProps } = BaseInputPropTypes
+const { error, type, autoComplete, placeholder, ...privateKeyProps } =
+  BaseInputPropTypes
```

Comment:
autoComplete is also set locally, so we need to omit it from the BaseInputProps. Same for `SeedPhraseInput`